.github/workflows/macos-ci.yml

```diff
@@ -0,0 +1,68 @@
+name: 'MacOS - CI'
```

Comment:
Please remove the spaces in the name. I find that these CI workflows work best when their internal name matches their filename.